index.html

```diff
@@ -7,227 +7,242 @@
-        don’t hestiate to contact me.
-      </p>
+      <div class="info-main">
+        <h1 class="info">Hey There.
+          I’m Habtamu </h1>
+          <h2>I’am a Software Developer</h2>
+          <p class="main-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+          </p>
+      </div>
       <ul class="icons">
         <li><a href="#"><img src="images/github.svg" alt=""></a></li>
         <li><a href="#"><img src="images/Linkedin.svg" alt=""></a></li>
         <li><a href="#"><img src="images/H.svg" alt=""></a></li>
         <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
         <li><a href="#"><img src="images/M.svg" alt=""></a></li>
       </ul>
+    </section>
     
    <!--  second project -->
-    </section>
-
-    <section class ="works">
-      <h2 class="section-header">My Recent Works</h3>
-        <hr>
+    <div class ="works">
+      <section class="works-head">
+        <h2 class="section-header">My Recent Works</h2>
+        <img class="line-horizontal" src="images/Vector@2x.png">
+      </section>
         <!-- project 1-->
-      <section class="projects" id ="pro-1">
+      <section class="projects1" id ="pro-1">
         <div class="img-wrapper">
-          <img class="placeholder" src="images/Img_Placeholder.png">
-        </div>
-        <h3 class="project-title">Multi-Post Stories</h3>
-        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text</p>
-         <ul class="languages">
-          <li class="langn">css</li>
-          <li class="langn">html</li>
-          <li class="langn">bootstrap</li>
-          <li class="langn" >Ruby</li>
-        </ul>
-        <button class="project-button">See Project</button>
-        
+          <div class="story-image">
+            <img class="placeholder" src="images/Img_Placeholder.png">
+          </div>
+          <div class="mult-story">
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="languages">
+              <li class="langn">css</li>
+              <li class="langn">html</li>
+              <li class="langn">bootstrap</li>
+              <li class="langn">Ruby</li>
+            </ul>
+            <button class="project-button">See Project</button>
+          </div>
+        </div>  
```

Comment:
[OPTIONAL]
- [ ] You may also consider using the same image in your cards as the ones provided on the template in Figma.